src/features/author.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+
+type State = {
+  author: User | null
+};
+
+const initialState: State | null = {
+  author: null,
+};
+
+const author = createSlice({
+  name: 'author',
+  initialState,
+  reducers: {
+    setUser: (state, action: PayloadAction<User | null>) => {
+      state.author = action.payload;
+
+      return state;
+    },
+  },
+});
+
+export default author.reducer;
+export const { actions } = author;
```

Comment:
The exported actions should be destructured to improve usability. Instead of `export const { actions } = author;`, you should use `export const { setUser } = author.actions;`